config/vufind/BlenderMappings.yaml

```diff
@@ -27,6 +27,7 @@
 # Ignore        true to ignore all values for a filter, or an array of values to
 #               ignore. Otherwise any filter value that does not have a field and
 #               value mapping causes the backend to be disabled for the search.
+# CountIgnored  Count ignored mappings to facet results. Default: true.
```

Comment:
I'd explain this in a bit more detail to make the purpose clear. Something like "If a filter is ignored, include the filter-less result count for the source in a facet. Should normally be left as true for expected results." Not really happy with that either, but perhaps @demiankatz can offer a better English explanation.

Alternatively we could leave this undocumented since I can't really come up with a scenaria where the old behavior makes sense.

Reply:
If we set this to false, would the number of results shown on the listed facet value be different from the number of results retrieved after applying the facet? If that's the case, I don't see any reason to make the behavior optional (except maybe debugging), so it would probably be okay to drop the setting entirely. If that's not what woud happen, then maybe I'm failing to understand what this is even for... :-)

Reply:
I can't think of a situation where one would not count the ignored facets, too. However, @EreMaijala mentioned in the email that it would be best to make it configurable, so i made it configurable. :D

Reply:
@GrothTobias Sorry, I'm obviously flip-flopping here. Let's _not_ make it configurable. I thought that there might have been a case for configuration, but since I can't really come up with anything meaningful, I have to admit that I just didn't think about it during the initial implementation.

Reply:
No need to worry. I just thought, that there might be some rare cases i can't think of. :)

Reply:
It's still possible, but we can always revisit this if it ever comes up. :)